test/openst/00_setup.js

```diff
@@ -0,0 +1,27 @@
+// 'use strict';
```

Comment:
The name of this file is different from all the other files. Suggest aligning it for developer happiness :-)

Reply:
Made it like this so that tests(npm run test) run in a logical order. 
e.g. First setup of openst, then create user wallet, then direct transfer etc.